components/contribution-flow/ContributionFlowStepsProgress.js

```diff
@@ -27,17 +27,19 @@ const StepLabel = styled(Span)`
   margin-bottom: 4px;
 `;
 
-const PrettyAmountFromStepDetails = ({ stepDetails, currency, isFreeTier }) => {
+const PrettyAmountFromStepDetails = ({ stepDetails, currency, isFreeTier, isCrypto }) => {
   if (stepDetails.amount) {
     const totalAmount = stepDetails.amount + (stepDetails.platformContribution || 0);
-    return (
+    return !isCrypto ? (
       <FormattedMoneyAmount
         interval={stepDetails.interval !== INTERVALS.flexible ? stepDetails.interval : null}
         currency={currency}
         amount={totalAmount}
         abbreviateInterval
         amountStyles={null}
       />
+    ) : (
+      <Span>{`${stepDetails.amount} ${stepDetails.currency.value}`}</Span>
```

Comment:
What's blocking to use FormattedMoneyAmount?

Reply:
@znarf : I wanted to keep things simple by not messing up the `FormattedMoneyAmount`. Note that `FormattedMoneyAmount` uses `Currency`, and that assumes we are passing the value in cents; i.e, I don't think we thought about crypto currencies when writing these components. Also note that the currency symbol library that we use `currency-symbol-map` doesn't have all the crypto currency symbols that we need; https://github.com/bengourley/currency-symbol-map/blob/master/map.js. For example I don't see `STORJ` on the list.

Reply:
But please feel free to let me know if you have suggestions or better alternatives. 🤔

Reply:
It's not that simpler because you're repeating it multiple times. Just make sure `FormattedMoneyAmount` or `Currency` return what you need when necessary. Detect early on that it's a crypto currency and return what's needed.

Reply:
@znarf : Makes sense, I've corrected that to include the logic to format the crypto currencies within `FormattedMoneyAmount`.